Content.Client/GameObjects/Components/Kitchen/MicrowaveBoundUserInterface.cs

```diff
@@ -285,7 +285,7 @@ public MicrowaveMenu(MicrowaveBoundUserInterface owner)
 
                 CookTimeInfoLabel = new Label
                 {
-                    Text = Loc.GetString("COOK TIME: 1"),
+                    Text = $"{Loc.GetString("microwave-menu-cook-time-label")} 1",
```

Comment:
Bit sketchy on this one.

Though I know of no language that doesn't accept the arabic numerals as at least a valid alternative, so it's probably fine?

Reply:
wasn't sure on this one, probably would be better to change the 1 into {$cookTime}